index.js

```diff
@@ -0,0 +1,126 @@
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const baseTripsURL = axios.defaults.baseURL = "https://trektravel.herokuapp.com/trips/"
+
+const loadTrips = () => {
+  const tripList = $("#trips-list");
+  const tripListContainer = $('#all-loc');
+  tripList.empty('');
+
+  axios.get(baseTripsURL)
+    .then((response) => {
+      response.data.forEach(location => {
+        tripList.append(`<li><a class="show-loc" href="${baseTripsURL}${location.id}">${location.name}</a></li>`)
+      })
+      
+    })
+    .catch((error) => {
+      console.log(error)
+      if (error.response.data && error.response.data.errors) {
```

Comment:
This should read:  `if (error.response && error.response.data && error.response.data.errrors) {`

That way for network errors it doesn't crash.